client/src/features/session/components/status/SessionHibernationStatusDetails.tsx

```diff
@@ -43,6 +62,16 @@ export default function SessionHibernationStatusDetails({
               noCaption
             />
           </p>
+          {hibernationCullTimestamp && (
+            <p className="mb-0">
+              <span className="fw-bold">Will be deleted:</span>{" "}
```

Comment:
To keep it consistent with the other suggestion
```suggestion
              <span className="fw-bold">Deleting in </span>{" "}
```

Reply:
Oh wait, is the time absolute here?